docs/mpcie_modules.md

```diff
@@ -207,6 +207,29 @@ can be done, for example in Debian OS, by performing those steps:
     sudo apt-get install firmware-realtek
     ```
 
+## SATA controllers
+
+### ASMedia ASM1061
+
+ASM1061 was tested with Delock 95233 mPCIe 2xSATA module.
+
+![ASM1061](images/delock_95233.png)
+
+**Compatible slots**
+
+Due to apu boards construction, it is possible to test that module only under
+apu2 mPCIe2 slot. Capacitors placed near the mPCIe1 slot prevent connecting the
+SATA cables to the module. Therefor, module was tested only in mPCIe2 slot.
```

Comment:
```suggestion
SATA cables to the module. Therefore, module was tested only in mPCIe2 slot.
```

Reply:
@piotr-kleins the typo left

Reply:
Weird, in changed files it is fixed. In my local repo also

Reply:
Ok there is another typo in another doc